airgun/views/virtwho_configure.py

```diff
@@ -218,6 +218,25 @@ def is_displayed(self):
                 and self.breadcrumb.read() != 'Create Config'
         )
 
+    @View.nested
+    class options(View):
+        status = Text("//strong[text()='Status']")
```

Comment:
I find this approach troublesome from theoretical point of view.

You want to confirm that labels are what we expect, without "Satellite Virt Who Configure/Config" part. But you select element for comparison based on label value. If value is wrong (bug exists), this approach will not find element at all. When using provided test, you will get confusing exception about KeyError instead of simple to understand string mismatch, along with current value and expected value. Moreover, if label ever change, you will have to modify Airgun instead of test - while when [oracle](https://glossary.istqb.org/en/search/oracle) gets outdated, we expect to update test instead of go hunting in frameworks code.

Following XPath is longer, but will match exact element you are looking for - no matter what label it currently has. Unless element disappeared (which would be another bug), label will be always present in `read()` results:
```python
status_label = Text("//span[contains(@class, 'config-hypervisor_id')]/../preceding-sibling::div/strong")
```
You can store it as template and `.format()` for each label to save yourself some typing:
```python
__label_template = "//span[contains(@class, '{class_name}')]/../preceding-sibling::div/strong"
status_label = Text(__label_template.format(class_name="config-hypervisor_id")
```

Reply:
oh, this method of the relative path is awesome, I knew mine is not good but could not come up with a better one, thanks.